### ISSUE TITLE
The sanitize_packages function prevents use of multiple local deb packages

### ISSUE THREAD

**Issue by davidchouse** [bdrung/bdebstrap#7]
**Steps to reproduce**: 
Add multiple local .deb packages, e.g. `/home/user/foo.deb` and `/home/user/bar.deb`, to the configuration and the resulting yaml file will only contain the last one due to the assumption that entries with a `/` are versioned.

**Problem Code**:
```python
        for package in self["mmdebstrap"]["packages"]:
            if not package:
                # Cover commented out and empty entries
                continue
            if "=" in package:
                name, version = package.split("=", 1)
                packages[name] = "=" + version
            elif "/" in package:
                name, version = package.split("/", 1)
                packages[name] = "/" + version
            else:
                packages[package] = ""
```

**Proposed solution**: 
Add a check to skip sanitization if the package name starts with `?`, `!`, `~`, `(`, `/`, `./`, or `../`. These were derived from the mmdebstrap documentation at: 
https://gitlab.mister-muffin.de/josch/mmdebstrap/wiki#:~:text=%3F%2C%20!%2C%20~%2C%20(%2C%20/%2C%20./%20or%20../

**Comment by bdrung** [comment]
Thanks for the bug report. That report is nicely troubleshooted and easy to convert into a test case.

I committed https://github.com/bdrung/bdebstrap/commit/1826acd948d470dd055e7a239b1487b025a09316 to fix `sanitize_packages`. Please test this version. Maybe the mmdebstrap call needs to be modified as well.

**Comment by bdrung** [comment]
Fixed in bdebstrap 0.6.0. Please re-open if commit 1826acd was not enough.
